tests/core/fs/test_fs_idempotent_sync.py

```diff
@@ -115,7 +115,8 @@ async def try_to_create_already_exists(self, type, path):
         @rule(path=BadPath)
         async def try_to_update_file(self, path):
             with pytest.raises(OSError):
-                await self.workspace.write_bytes(path, data=b"a", offset=0)
+                async with await self.workspace.open_file(path, "rb+"):
```

Comment:
`+` is not documented in the `WorkspaceFile` docstring

Reply:
@Sithi5 Could you fix the docstring?

Reply:
Yes, done